doc/pages/blurbs.json

```diff
@@ -324,5 +324,14 @@
     "link_name": "jenniferwiner.com",
     "link": "https://www.jenniferwiner.com",
     "social": "globe"
+  },
+  {
+    "name": "John Maguire",
+    "image": "/assets/contributors/john_maguire.jpg",
```

Comment:
Looks like you need to rename headshot.png to john_maguire.jpg

Reply:
:man_facepalming: resized the image but forgot to rename